specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -2637,12 +2696,12 @@
       }
     },
     "PrivateEndpointConnectionListResult": {
-      "type": "object",
       "description": "List of private endpoint connection associated with the specified workspace",
+      "type": "object",
       "properties": {
         "value": {
-          "type": "array",
           "description": "Array of private endpoint connections",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/PrivateEndpointConnection"
```

Comment:
Does this conform to the common-types privateEndpointConnection definition? Can it be changed to a $ref? 

[https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/privatelinks.json#L20](https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/privatelinks.json#L20)

Reply:
I double-checked the code side there are more properties expected compared to the common schema, so I will keep this for now just in case we break anything. I will create a task to see if we can migrate to common schema in the next version.

Reply:
https://msdata.visualstudio.com/Vienna/_workitems/edit/2688199